docs/reproduce/msmarco-v1-passage.html

```diff
@@ -218,18 +218,18 @@ <h1 class="mb-3">MS MARCO V1 Passage</h1>
   -threads 16 \
   -index msmarco-v1-passage \
   -topics dl19-passage \
-  -output runs/run.msmarco.bm25-default.dl19.txt \
+  -output runs/run.msmarco.bm25.dl19.txt \
   -hits 1000 -bm25
 </code></pre></blockquote>
 Evaluation commands:
 
   <blockquote class="mycode">
-<pre><code>tools/eval/trec_eval.9.0.4/trec_eval -c -l 2 -m map dl19-passage \
-  runs/run.msmarco.bm25-default.dl19.txt
+<pre><code>tools/eval/trec_eval.9.0.4/trec_eval null dl19-passage \
+  runs/run.msmarco.bm25.dl19.txt
 tools/eval/trec_eval.9.0.4/trec_eval -c -m ndcg_cut.10 dl19-passage \
-  runs/run.msmarco.bm25-default.dl19.txt
-tools/eval/trec_eval.9.0.4/trec_eval -c -l 2 -m recall.1000 dl19-passage \
-  runs/run.msmarco.bm25-default.dl19.txt
+  runs/run.msmarco.bm25.dl19.txt
+tools/eval/trec_eval.9.0.4/trec_eval null dl19-passage \
```

Comment:
why null here?

Reply:
Not sure what happened here, but rebuilding solved it.